pan-domain-auth-play/src/main/scala/com/gu/pandomainauth/action/Actions.scala

```diff
@@ -33,10 +33,10 @@ trait AuthActions {
     */
   def wsClient: WSClient
   def controllerComponents: ControllerComponents
-  def panDomainSettings: PanDomainAuthSettingsRefresher
+  val panDomainSettings: PanDomainAuthSettingsRefresher
```

Comment:
This is the core of the fix - we don't want to repeatedly evaluate `panDomainSettings` and get a new one each time - so it shouldn't be a `def`!